CHANGELOG.md

```diff
@@ -5,12 +5,16 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
-### Added
-- **CUMULUS-491** - Add granule reconciliation API endpoints.
 
 ### Added
+- **CUMULUS-491** - Add granule reconciliation API endpoints.
 - **CUMULUS-508** - `@cumulus/deployment` cloudformation template allows for lambdas and ECS clusters to have multiple AZ availability.
     - `@cumulus/deployment` also ensures docker uses `devicemapper` storage driver.
+- **CUMULUS-755** - Add DynamoDB autoscaling support
```

Comment:
Seems like we will need some documentation on this on cumulus-nasa.github.io

Reply:
Agree, will open a PR

Reply:
Although, I'm going to wait on opening a PR until I'm certain we are going with a given implementation for configuring the autoscaling options